ego/premain/core/core.go

```diff
@@ -46,9 +46,16 @@ type Mounter interface {
 }
 
 // PreMain runs before the App's actual main routine and initializes the EGo enclave.
-func PreMain(payload string, mounter Mounter, fs afero.Fs) error {
+func PreMain(payload string, mounter Mounter, fs afero.Fs, originalEnviron []string) error {
```

Comment:
```suggestion
func PreMain(payload string, mounter Mounter, fs afero.Fs, hostEnviron []string) error {
```
isn't that a better name?